README.md

```diff
@@ -174,6 +174,63 @@ var solidus_client = new SolidusClient({resources_options: resources_options});
 solidus_client.resources_options = resources_options;
 ```
 
+# Utilities
+
+## Assets Proxy
+
+This library can be used both server and client side, to replace assets URLs with proxy URLs. For example, all strings containing URLs starting with `http://resource1.com/images` in a context could be replaced to `http://mysite.proxy.com/resource1/...` in a preprocessor.
+
+### Initialization
+
+```javascript
+var AssetsProxy = require('solidus-client/lib/assets_proxy');
+var assets_to_proxy = [
+  {
+    resource: 'http://resource1.com/images',
```

Comment:
What are your thoughts on a different name for this key? When I was first reading this, I was confused because I was thinking of Solidus resources, but we're essentially replacing something within Solidus resources. What about a name like `origin`?